Slider: update demo description (#2069)
DevExpress/devextreme-demos: JSDemos/Demos/Slider/Overview/Angular/description.md

> The Slider component allows users to set a numeric value on a continuous range of possible values. This demo shows how you can create and configure a Slider.

Reviewer: Right now, these two sentences don't really flow one from another. There's nothing in the second sentence that refers to the first really. They might be two separate paragraphs, and that would probably read a bit better. 

```suggestion
The Slider component allows users to set a numeric value on a continuous range of possible values. 

This demo shows how you can create and configure a Slider.
```

Another thing to think about is whether you need a Slider definition at all. For example, this is how Microsoft describes a Track Bar in WinForms: "Represents a standard Windows track bar." They don't list UI element that comprise the control or possible user actions. A track bar is considered to be simple and universal enough to be understood by most developers. 

In this case, we show plenty of Slider examples right here in this demo. So again, I would maybe only leave a textual description if I were thinking of search engine optimization... But I don't think that the current description would help much. 

In other words, you can go without the first sentence. 

```suggestion
This demo shows how you can create and configure a Slider.
```

Finally, you can reverse the order of the sentences to make the text flow more natural. You will mention "a Slider". Then the natural question is "what is a slider"? And then you answer it. 

```suggestion
This demo shows how you can create and configure a Slider. This component allows users to set a numeric value on a continuous range of possible values. 
```

(My preference is to leave just one sentence.)

## Create a Slider

To create a Slider, declare it in markup and use the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) properties to limit the range of accepted values.

You can also specify the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property to change the initial value.

> You can see the resulting Slider in the **Default mode** section of this demo.

Reviewer: I wouldn't use the phrase "resulting Slider", because you didn't really give specific instructions. Here's my version:

```suggestion
The **Default mode** section shows a Slider with such a basic setup.
```

> ## Customize Slider Appearance

Reviewer: It's a lot of information in one section. Some features require lots of text. Others you have listed in the same paragraph ("step" and "disabled"). Maybe it will be a bit better if we break this down into smaller sections, each corresponding to a slider components in the demo. The sections could be as follows (labels in the demo would need to be updated as well): 

## Max and min labels
## Handle tooltip
## No range highlight
## Discrete step
## Disabled state

> The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, enable the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible) property to display the labels. You can also specify the [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position) and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.

Reviewer: I first noticed two similar phrases in consecutive sentences: "To configure the labels", "to display the labels". If I want to configure them, I'll probably display them. So that's one reason I wouldn't want to say the second one. In addition to that, the name `visible` is self-explanatory. So I want to cut some text. Maybe use this:

```suggestion
The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, set the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible), [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position), and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.
```

> The Slider can also display a tooltip for a handle. To display a tooltip, you need to configure the [tooltip](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/) object:

Reviewer: If you will go with section titles, as I suggested above, then you wouldn't need the first sentence: 

```suggestion
To display a handle tooltip, you need to configure the [tooltip](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/) object:
```

- Set the [enabled](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#enabled) property to `true` to display a tooltip. 

> - Specify the tooltip [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#position), over or under a Slider.

Reviewer: A colon is more appropriate than a comma here - you expand on a previous statement.

```suggestion
- Specify the tooltip [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#position): over or under the Slider.
```

- Specify the [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#format) property.

- Assign *'onHover'* or *'always'* to the [showMode](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#showMode) property to change how the component shows a tooltip.

> Use the [showRange](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#showRange) property to specify if the selected range should be highlighted. You can also use the [step](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#step) property to specify the value change step for the Slider.

Reviewer: The phrase "selected range" would be more appropriate if you actually had a range bar (two handles). A slider simply sets a value - I don't think of it as a range. 

(These paragraphs below would have their captions too...)

```suggestion
Use the [showRange](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#showRange) property to specify if the component should highlight the rangle between [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value). 

You can use the [step](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#step) property to specify the value change step for the Slider.
```

If you want to disable the Slider, set the [disabled](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#disabled) property to `true`.

## Handle the Value Change Event

> To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#value) property of the [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) component to the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property of the RangeSlider.

Reviewer: A couple of notes here: 

- It seems like a copy-paste from some other demo (RangeSlider reference)
- NumberBox reference seems confusing. Looks like I have to use a NumberBox to handle value changes in a Slider. 

```suggestion
To handle value changes, use two-way binding for the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property. In this demo, two sliders are bound to a [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) component.
```

> The [callValueChange](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#callValueChange) property allows you to choose when to change the Slider value and supports the `onMoving` and `onMovingComplete` modes. You can see the property's effect in the **Process Value Changes** section. The first Slider changes its value every time a user slides the handle. The second Slider changes its value only when the handle is released.

Reviewer: "allows you to choose when to change" - seems like I'm choosing, and I'm also changing. But in this case the control is changing the value, so I wanted to make that clear. 

I also split the first sentence into two. 

The last two sentences: 

- "while" explains the behavior better than "every time"
-  "is released" - just wanted to get rid of passive voice

```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#callValueChange) property allows you to choose when the Slider changes its value. Available modes include `onMoving` and `onMovingComplete`. You can see the property's effect in the **Process Value Changes** section. The first Slider changes its value while a user slides the handle. The second Slider changes its value only after the user releases the handle.
```

## Create a Slider

To create a Slider, declare it in markup and use the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) properties to limit the range of accepted values.

You can also specify the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property to change the initial value.

The **Default mode** section shows a Slider with such a basic setup.

## Customize Slider Appearance

### Max and min labels

The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, specify the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible), [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position), and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.

> ### Handle tooltip

Reviewer: Now that I read this, it seems a bit confusing - I first assume that "handle" is a verb. So maybe change this caption to simply "Tooltips"? 

```suggestion
### Tooltips
```

To display a handle tooltip, you need to configure the [tooltip](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/) object:

- Set the [enabled](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#enabled) property to `true` to display a tooltip. 

- Specify the tooltip [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#position): over or under the Slider.

- Specify the [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#format) property.

- Assign *'onHover'* or *'always'* to the [showMode](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/tooltip/#showMode) property to change how the component shows a tooltip.

> ### No range highlight

Reviewer: Again - I know I suggested this one, but now it doesn't look great to me, sorry. I would just say "Range highlight". The meaning is pretty much the same, but the phrase is easier to understand. 

```suggestion
### Range highlight
```
